### PR TITLE
Add South Africa configs

### PR DESCRIPTION
**Pull request by Matt-Yorkley**
Add configs for the new/updated South Africa instance.

### CODE REVIEW

File: inventory/host_vars/ofn.kandu.co.za/config.yml [hunk 1]
```diff
@@ -1,6 +0,0 @@
----
-
-domain: ofn.kandu.co.za
```
Comment (mkllnk) [inline inventory/host_vars/ofn.kandu.co.za/config.yml]: That server still exists. What's happening with it? Is there a handover, is Kandu still alive or do we have two instances now?

File: inventory/host_vars/www.openfoodnetwork.org.za/config.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+rails_env: production
+
+mail_domain: openfoodnetwork.org.za
+
+certbot_domains:
+  - www.openfoodnetwork.org.za
+  - openfoodnetwork.org.za
+
+unicorn_workers: 2
+swapfile_size: 1G
```
Comment (mkllnk) [inline inventory/host_vars/www.openfoodnetwork.org.za/config.yml]: I'm wondering if we should make this the standard now.

File: inventory/host_vars/ofn.kandu.co.za/config.yml (hunk 1 above)
Comment (Matt-Yorkley) [reply inventory/host_vars/ofn.kandu.co.za/config.yml]: They're basically migrating to the new domain, and ofn-install isn't really used on the existing server anyway.

File: inventory/host_vars/www.openfoodnetwork.org.za/config.yml (hunk 2 above)
Comment (Matt-Yorkley) [reply inventory/host_vars/www.openfoodnetwork.org.za/config.yml]: I went for some conservative options as I'm not sure about their specs, but yes, we can probably set some bigger defaults soon for all servers.

**Comment by sauloperez** [comment]
happy with the answers @mkllnk ?

**Pull request merged**